packages/output/src/utils/getTextElementTagNames.js

```diff
@@ -37,9 +38,24 @@ function getTextElementTagNames(elements) {
        * @param {string} element.id Element ID
        * @param {number} element.fontSize Font size.
        * @param {string} element.content Text content.
+       * @param {string} element.tagName Existing tagName
        * @return {Map<string, string>} Tag names map.
        */
-      (tagNamesMap, { id, fontSize, content }) => {
+      (tagNamesMap, { id, fontSize, content, tagName }) => {
+        // if a new tagName is added,
+        // add it to the element
+        if (newTag) {
+          tagNamesMap.set(id, newTag);
+          return tagNamesMap;
+        }
+
+        // if a tag already exists,
+        // utilize the one already part of the element
+        if (tagName) {
```

Comment:
If a user chooses "Automatic" from the dropdown, we set `tagName = 'auto'`, on the element. We don't wanna actually set that as a tag name though.

```suggestion
        if (tagName && tagName !== 'auto') {
```